api/RTCIceCandidateType.json

```diff
@@ -0,0 +1,60 @@
+{
+  "api": {
+    "RTCIceCandidateType": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCIceCandidateType",
+        "support": {
+          "chrome": {
+            "version_added": false,
+            "notes": "An <code>RTCIceCandidate</code> object's <code>type</code> is not maintained by this browser."
+          },
+          "chrome_android": {
+            "version_added": false,
+            "notes": "An <code>RTCIceCandidate</code> object's <code>type</code> is not maintained by this browser."
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": false,
+            "notes": "An <code>RTCIceCandidate</code> object's <code>type</code> is not maintained by this browser."
+          },
+          "firefox_android": {
+            "version_added": false,
+            "notes": "An <code>RTCIceCandidate</code> object's <code>type</code> is not maintained by this browser."
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": null
+          },
+          "opera_android": {
+            "version_added": null
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          },
+          "webview_android": {
+            "version_added": false,
+            "notes": "An <code>RTCIceCandidate</code> object's <code>type</code> is not maintained by this browser."
+          }
+        },
+        "status": {
+          "experimental": false,
```

Comment:
Does any browser implement this? I'm not sure we should say it's stable if no browser has shipped it yet.

Reply:
Well, stability is about the spec, not about implementation status.

Reply:
OK!